Dockerfile

```diff
@@ -813,4 +813,78 @@ USER nextjs
 # https://nextjs.org/docs/pages/api-reference/next-config-js/output
 CMD ["node", "apps/vpnmanager/server.js"]
 
+# ============================================================================
+# Techlab Blog
+# ============================================================================
+
+#
+# techlabblog-deps: image with all techlabblog dependencies
+# -----------------------------------------------------
+
+FROM base-deps AS techlabblog-deps
+
+COPY apps/techlabblog/package.json ./apps/techlabblog/package.json
+
+# Use virtual store: https://pnpm.io/cli/fetch#usage-scenario
+RUN pnpm --filter "./apps/techlabblog" install --offline --frozen-lockfile
+
+#
+# techlabblog-builder: image that uses deps to build shippable output
+# ------------------------------------------------------------------
+
+FROM base-builder AS techlabblog-builder
+
+ARG NEXT_TELEMETRY_DISABLED \
+  # Next.js / Payload (build time)
+  PORT \
+  # Next.js (runtime)
+  NEXT_PUBLIC_APP_NAME="Techlab Blog" \
+  NEXT_PUBLIC_APP_URL \
+  NEXT_PUBLIC_SENTRY_DSN \
+  NEXT_PUBLIC_SEO_DISABLED \
+  NEXT_PUBLIC_GOOGLE_ANALYTICS \
+  # Sentry (build time)
+  SENTRY_AUTH_TOKEN \
+  SENTRY_ENVIRONMENT \
+  SENTRY_ORG \
+  SENTRY_PROJECT
+
```

Comment:
I don't see where these arguments are referenced in the build

Reply:
@koechkevin Not sure I follow. Can you clarify on this?